### PR TITLE
introduce flake8 to make codes pep8

### PR DESCRIPTION
**Pull request by zsluedem**
## Overview
introduce flake8 to make codes pep8

### JIRA ticket:
https://rchain.atlassian.net/browse/RCHAIN-3659

### Notes
<sup>_Optional. Add any notes on caveats, approaches you tried that didn't work, or anything else._</sup>

### Please make sure that this PR:
- [x] is at most 200 lines of code (excluding tests),
- [x] meets [RChain development coding standards](https://rchain.atlassian.net/wiki/spaces/DOC/pages/28082177/Coding+Standards),
- [x] includes tests for all added features,
- [x] has a reviewer assigned,
- [x] has [all commits signed](https://rchain.atlassian.net/wiki/spaces/DOC/pages/498630673/How+to+sign+commits+to+rchain+rchain).

### [Bors](https://bors.tech/) cheat-sheet:

- `bors r+` runs integration tests and merges the PR (if it's approved),
- `bors try` runs integration tests for the PR,
- `bors delegate+` enables non-maintainer PR authors to run the above.

### CODE REVIEW

**Comment by zsluedem** [comment]
bors try

**Review by KentShikama — APPROVED** [review]

File: integration-tests/test/rnode.py [hunk 1]
```diff
@@ -523,10 +524,10 @@ def make_bootstrap_node(
     ])
 
     container_command_options = {
-        "--port":                   40400,
-        "--validator-private-key":  keypair.private_key,
-        "--validator-public-key":   keypair.public_key,
-        "--host":                   container_name,
```
Comment (adaszko) [inline integration-tests/test/rnode.py:529]: This tool is making the code less readable.  Is there a way to disable particular formatting hint?

**Review by adaszko — CHANGES_REQUESTED** [review]

**Comment by adaszko** [comment]
Is it really worth it to introduce this?  The only reason I mentioned PEP8 was some lack of few blank lines in imports, which is a trifle.  Other tools like `pylint` and `mypy` actually point out potential bugs, this one just points out formatting issues.  The downside is that you can now do `bors try`, come back after an hour or more only to find the build has failed on some missing whitespace.  Consider perhaps integrating an automated code formatter with your editor instead.  One such tool which is popular is [Black](https://github.com/python/black).

File: integration-tests/test/rnode.py (hunk 1 above)
Comment (zsluedem) [reply integration-tests/test/rnode.py:529]: https://www.python.org/dev/peps/pep-0008/#pet-peeves that's pep8 suggest.

**Comment by zsluedem** [comment]
OK~! Close it~

**Pull request closed without merging**